### DISCUSSION TITLE
Protected Session

### DISCUSSION THREAD

**Discussion by rjkantor** [TriliumNext/Trilium#2533] [General]
Hello I am new to Trilium 0.49.3.   I have installed the MacOS version and am trying to protect subtree, I enter a password, and then it says wrong password.  Is there some default or something else which may be causing this error.

I have added my own items to the preexisting pages.

Any input on correcting, is appreciated.

I am running on an M1 MacBook Pro, could the issue be due to the app being Intel native?

Robb

**Comment by zadam** [comment]
Hi, as part of the initial setup you were asked to enter the password:

![image](https://user-images.githubusercontent.com/617641/148659775-717e8088-282c-481e-b96d-9aee8e709606.png)

This is the password you need to enter when protecting the notes/subtree.

**Reply by rjkantor** [reply]
Thank you for the reminder.  That worked.  I appreciate the quick response.  I look forward to further enhancements.

Robb
